src/shader_recompiler/frontend/translate/vector_alu.cpp

```diff
@@ -1046,11 +1033,9 @@ void Translator::V_SAD_U32(const GcnInst& inst) {
 void Translator::V_LSHL_B64(const GcnInst& inst) {
     const IR::U64 src0{GetSrc64(inst.src[0])};
     const IR::U64 src1{GetSrc64(inst.src[1])};
-    const IR::VectorReg dst_reg{inst.dst[0].code};
     ASSERT_MSG(src0.IsImmediate() && src0.U64() == 0 && src1.IsImmediate() && src1.U64() == 0,
                "V_LSHL_B64 with non-zero src0 or src1 is not supported");
-    ir.SetVectorReg(dst_reg, ir.Imm32(0));
-    ir.SetVectorReg(dst_reg + 1, ir.Imm32(0));
+    SetDst64(inst.dst[0], ir.Imm64(0LL));
```

Comment:
Best to keep this as is to avoid redundant pack instruction

Reply:
Reverted this one.